code/modules/mob/living/carbon/human/human.dm

```diff
@@ -966,10 +966,12 @@
 		to_chat(target, span_warning("You can't piggyback ride [src] right now!"))
 		return
 
+	//SKYRAT EDIT START
 	visible_message(span_notice("[target] starts to climb onto [src]..."))
```

Comment:
```suggestion
	visible_message(span_notice("[target] starts to climb onto [src]..."))
```

Reply:
Don't forget this one too, as the edit start isn't necessary for something that is changing the value of a single line